GoogleForCreators/web-stories-wp
# Storybook: Move Menu Out of DropDown

## Summary
En route to building a typeahead and popover menu, extracting menu from dropDown to be reused. 

## Relevant Technical Choices

- Moving sampleData out of dropDown and into both testUtils and storybookUtils so that it can change if we need specific test cases without impacting storybook. 
- Adding a storybook section for just menu.
- Renaming `selectButtonId` prop in menu to `parentId`

## To-do



## User-facing changes

None

## Testing Instructions

Verify that the dropDown in storybook/DesignSystem still works as expected
See that there's now a storybook/DesignSystem/components/Menu that's just the menu.

---



Fixes #
